tests/unit/test-cmd/test.run.js

```diff
@@ -36,6 +36,7 @@ describe('run', () => {
       artifactsDir: path.join(sourceDir, 'web-ext-artifacts'),
       sourceDir,
       noReload: true,
+      jsconsole: true,
```

Comment:
This should not be true by default. In fact, you should probably remove it to let the test invoke the default value.